### PR TITLE
Setting up class with pixi

### PR DESCRIPTION
**Pull request by Hofer-Julian**
- Replaces `environment.yml` and `parallel_class_environment_setup.sh` with `pixi.toml`
- Adds pixi binaries for `aarch-apple` and `x64-linux`
- Adds pixi wrapper which uses the correct binary
- Adjust scripts and docs accordingly

### CODE REVIEW

File: wsl_setup.md [hunk 1]
```diff
@@ -20,30 +20,18 @@ Clone the class repo in your WSL Ubuntu virtual machine after opening a WSL term
...
 
-## Installing the class `mf6xtd` conda environment
+## Setting up parallel MODFLOW 6 and pixi environment
 
-Open the WLS terminal and navigate to the `/parallel-modflow-dsd23/installation` directory. 
-
-### Clean WSL Ubuntu virtual environment, Linux, or MacOS
-
-If you installing the class conda environment in a WSL Ubuntu virtual environment, Linux machine, or MacOS machine without a Conda or Miniconda installation, the environment can be installed using
+In order to setting up you system execute:
```
Comment (christianlangevin) [inline wsl_setup.md]:
```suggestion
In order to set up parallel MODFLOW 6,  execute:
```

**Review by christianlangevin — COMMENTED** [review]
I like the looks of this.  Two small things: (1) do we want to put the large unversionable pixi binaries in the repo?  (2) should we preserve the conda approach somehow, perhaps as a fallback approach for those wanting to use conda?

**Comment by Hofer-Julian** [comment]
> do we want to put the large unversionable pixi binaries in the repo?

I agree that this is bad in most situations.
For a training I like it to be honest. It removes one variable which can go wrong.

> should we preserve the conda approach somehow, perhaps as a fallback approach for those wanting to use conda?

I don't mind either way.
Up to your preference :)

**Comment by mjr-deltares** [comment]
Thanks @Hofer-Julian for this PR! I'd like to discuss with @jdhughes-usgs before pushing the button (since he's done most of the work on the conda route). In the mean time, I will test it on a fresh Ubuntu/WSL install.

**Comment by mjr-deltares** [comment]
This all seems to be working smoothly on a fresh WSL/Ubuntu, let's merge!

**Pull request merged**